app/components/utils/mandatee-selector.js

```diff
@@ -33,26 +40,32 @@ export default class MandateeSelector extends Component {
       yield this.initialLoad;
     }
     const queryOptions = {
-      ...this.defaultQueryOptions,  // clone
+      ...this.defaultQueryOptions, // clone
     };
     if (searchTerm) {
       queryOptions['filter[person][last-name]'] = searchTerm;
     }
-    queryOptions['filter[government-body][:uri:]'] = CURRENT_GOVERNMENT_BODY;
+    queryOptions['filter[:lte:start]'] =
+      this.governmentBodyOfDate.toISOString();
```

Comment:
**Possibly** this could create an issue in some edge cases where the meeting is on the last/first day of a new government composition if the Date locale is not the same. (That's why I suggested using Date.UTC()). 

However, that'll be easily remedied by KAS-3344 when they can select **all** mandataries, so let's leave this like that.